packages/frontend-core/src/components/grid/controls/MigrationModal.svelte

```diff
@@ -0,0 +1,69 @@
+<script>
+  import {
+    ModalContent,
+    notifications,
+    Input,
+    InlineAlert,
+  } from "@budibase/bbui"
+  import { getContext } from "svelte"
+  import { ValidColumnNameRegex } from "@budibase/shared-core"
+
+  const { API, definition, rows } = getContext("grid")
+
+  export let column
+
+  let newColumnName = `${column.schema.name} migrated`
+  $: error = checkNewColumnName(newColumnName)
+
+  const checkNewColumnName = newColumnName => {
```

Comment:
Do we need to check the new column name against the other columns in the table? Just in case you chose a column name that's the same as another column in the table.

Reply:
Great call, done!